org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserInitiatedExceptionKey.java

```diff
@@ -0,0 +1,17 @@
+package org.code.javabuilder;
+
+/** These keys map to client-side keys that are translatable. Examples of what the strings */
```

Comment:
Here's a [brief doc](https://docs.google.com/document/d/1gBWh1b9rH1DRpheKph1I1KjT-melx01yB0xXs6g2GTU/edit#) describing our initial thoughts on how to design this. Most of the "why" comes from a [discussion with the infra team](https://codedotorg.slack.com/archives/CFTFD6BPV/p1617735713115200) (also linked in the doc).

There's a bit of between the lines context that I don't think was ever enumerated there. Building and maintaining the internationalization system is a big task. In general, we want to keep all that in one place. Rather than integrating Javabuilder with crowdin, it's easier to find a way to leverage the system we already have in place and keep all the internationalization in the CDO repo.